code/datums/traits/negative.dm

```diff
@@ -13,6 +13,17 @@
 
 
 
+/datum/trait/morerobustthanyou
+	name = "Brain Tumor"
+	desc = "You got a little friend in yo head thats drivin you insane in the membrane."
+	value = -2
+	mob_trait = TRAIT_HEAVY_SLEEPER
+	gain_text = "<span class='danger'>You feel smooth.</span>"
+	lose_text = "<span class='notice'>You feel wrinkled again.</span>"
+
+/datum/trait/morerobustthanyou/on_process()
+	adjustBrainLoss(1)
```

Comment:
stop calling procs on the wrong datums you chumbis

Reply:
Oh lol